src/io/flutter/perf/EditorPerfDecorations.java

```diff
@@ -267,13 +265,6 @@ public void clear() {
  * required.
  */
 class PerfGutterIconRenderer extends GutterIconRenderer {
-  static final AnimatedIcon RED_PROGRESS = new RedProgress();
```

Comment:
Moved this over to an Icons class as the Icons are now reused here and in the table.